components/notes.py

```diff
@@ -7,7 +7,10 @@ class Notes:
     @staticmethod
     def get_component():
         widget = dbc.Col(
-            style={"max-width": "40%"},
+            style={
+                "width": "40%",
```

Comment:
Can be replaced with `className="col-4"` or another bootstrap class like that :)

Reply:
Still ^